financial/client.go

```diff
@@ -0,0 +1,43 @@
+package financial
+
+import (
+	"github.com/checkout/checkout-sdk-go/client"
+	"github.com/checkout/checkout-sdk-go/common"
+	"github.com/checkout/checkout-sdk-go/configuration"
+)
+
+type Client struct {
+	configuration *configuration.Configuration
+	apiClient     client.HttpClient
+}
+
+func NewClient(configuration *configuration.Configuration, apiClient client.HttpClient) *Client {
+	return &Client{
+		configuration: configuration,
+		apiClient:     apiClient,
+	}
+}
+
+func (c *Client) GetFinancialActions(query QueryFilter) (*QueryResponse, error) {
```

Comment:
Additional method without parameters?

Reply:
Probably since this filter contains more properties, makes sense if the customer sends a new instance maybe we should check a good pattern for this kind of stuff